docker-compose.yml

```diff
@@ -3,6 +3,30 @@ version: "3.8"
 # TODO: Containerise frontend as well
 
 services:
+  web:
+    build: ./frontend/charity-management-app
```

Comment:
I think you accidently ran `npm create vite@latest` inside of the frontend folder, which then created another folder with the actual frontend inside of it called charity-management-app. Instead, we want the `frontend` directory to be the root of the directory.